sonar-csharp-plugin/pom.xml

```diff
@@ -63,7 +63,7 @@
   </scm>
 
   <properties>
-    <sonar.version>6.7</sonar.version>
+    <sonar.version>7.3-alpha1</sonar.version>
```

Comment:
I would add a comment here to say this is the development API dependency version...

Reply:
changed to 7.3